src/app/backend/dashboard.go

```diff
@@ -86,7 +89,15 @@ func main() {
 	// TODO(maciaszczykm): Move to /appConfig.json as it was discussed in #640.
 	http.Handle("/api/appConfig.json", handler.AppHandler(handler.ConfigHandler))
 	http.Handle("/metrics", prometheus.Handler())
-	log.Print(http.ListenAndServe(fmt.Sprintf("%s:%d", *argBindAddress, *argPort), nil))
+
+	// Listen for http and https
+	addr := fmt.Sprintf("%s:%d", *argBindAddress, *argPort)
+	go log.Fatal(http.ListenAndServe(addr, nil))
+	secureAddr := fmt.Sprintf("%s:%d", *argBindAddress, *argSecurePort)
+	if len(*argCertFile) != 0 && len(*argKeyFile) != 0 {
+		go log.Fatal(http.ListenAndServeTLS(secureAddr, *argCertFile, *argKeyFile, nil))
```

Comment:
@cheld I did not find a trivial way to implement it, it might be overkill to listen for a file change here.